src/app/models/form.server.model.ts

```diff
@@ -475,8 +469,16 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     return this.save()
   }
 
+  const FormDocumentSchema = (FormSchema as unknown) as Schema<IFormDocument>
```

Comment:
we don't need the `as unknown` in between right? I tried removing it and it still compiles. also could you pls add in a comment explaining the context for this and why it's necessary? would be good for other devs in future